src/models/SubjectIdentitiesModel.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021, IBM Deutschland GmbH
+ */
+import { Pool } from 'pg';
+import { Logger } from '@overnightjs/logger';
+import DB from '../server/DB';
+
+export class SubjectIdentitiesModel {
+
+    /**
+     * Verify if participant exists in database.
+     * @param subjectID The participant id
+     */
+    public async getSubjectIdentityExistence(subjectID: string): Promise<boolean> {
+        try {
+            const pool: Pool = DB.getPool();
+            const res = await pool.query('select * from studyparticipant where subject_id = $1', [
+                subjectID
+            ]);
+
+            if (res.rows.length !== 1) {
+                return false
+            }
+            return true;
+        } catch (err) {
+            Logger.Err(err);
+            throw err;
+        }
+    }
+
+    /**
+     * Add a new participant
+     * @param subjectID The participant id
+     */
+    public async addNewSubjectIdentity(subjectID: string): Promise<void> {
+        try {
+            const pool: Pool = DB.getPool();
+            await pool.query("INSERT INTO studyparticipant(subject_id) VALUES ($1);", [subjectID]);
```

Comment:
Could you replace double quotes with single quotes?

Reply:
done